openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/OpenAPIResourceMapper.java

```diff
@@ -210,10 +210,14 @@ private void generatePathItem(String httpMethod, Paths path, Operation operation
         //Add path parameters if in path and query parameters
         OpenAPIParameterMapper openAPIParameterMapper = new OpenAPIParameterMapper(resource, op, apiDocs);
         openAPIParameterMapper.getResourceInputs(components, semanticModel);
-        if (!openAPIParameterMapper.getErrors().isEmpty()) {
+        if ((!openAPIParameterMapper.getErrors().isEmpty() && openAPIParameterMapper.getErrors().size() > 1)
+                || (openAPIParameterMapper.getErrors().size() == 1 &&
+                !openAPIParameterMapper.getErrors().get(0).getCode().equals("OAS_CONVERTOR_113"))) {
```

Comment:
It will skip the adding relevant operation to YAML content, that time it will return an empty value :). If it is `OAS_CONVERTOR_113` we will add the operation details to YAML.